horizon_autotests/steps/volumes.py

```diff
@@ -384,7 +384,7 @@ def tab_backups(self):
 
     @pom.timeit('Step')
     def create_snapshot(self, volume_name, snapshot_name, description=None,
-                        check=True):
+                        check=True, modal_absent=True):
```

Comment:
I'm inclined to believe, that `modal_absent` and `check` have the same value here. If `check=False`, not need to worry that modal is absent too. How do you think?